shared/src/email/EmailParamsByEmailType.ts

```diff
@@ -11,6 +11,15 @@ import { AppellationLabel } from "../romeAndAppellationDtos/romeAndAppellation.d
 import { SiretDto } from "../siret/siret";
 import { Email } from "./email.dto";
 
+export type UserParamsForMail = {
```

Comment:
on peut expliciter un peu le nommage :
```suggestion
export type UserParamsForAgencyActivatedEmail = {
```

Reply:
Le problème c'est que c'est pas utilisé pour un mail spécifique on l'utilise partout ou on affiche le block d'info utilisateur

Reply:
ok alors on garde